docs/guides/refind.md

```diff
@@ -149,6 +149,14 @@ By default, rEFInd boots Linux indirectly by booting GRUB, systemd-boot etc. But
 
 8. Now, when you shall be in rEFInd, it should show an entry with the path of the image of your kernel and shall most probably have the icon of the Linux Penguin. That entry shall be the one which shall boot Linux directly using rEFInd.
 
+9. If you want to use your distro's icon instead of Linux Penguin one, you may label the volume containing your kernel with the name of your distro. For example if the distro is **Ubuntu** and kernel is in `/dev/nvme0n1p3` partition, run:-
+
+    ```plain
+    sudo tune2fs -L Ubuntu /dev/nvme0n1p3
```

Comment:
I think this'll only work on ext2/3/4 filesystems, some people may be using btrfs, zfs, etc.

Reply:
That's why I added the link to rEFInd's official guide

Reply:
`sudo btrfs filesystem label <mountpoint> <newlabel>`

Is this for btrfs?

Reply:
I probably can add instructions to label common fs in use?

Reply:
> `sudo btrfs filesystem label <mountpoint> <newlabel>`
> 
> 
> 
> Is this for btrfs?

That looks right but I can't check right now.

Reply:
Any other common FS @Redecorating?

Reply:
I cannot format to zfs BTW for testing, nor do I know any command that does so. But I believe that its not used that much anyways.

Reply:
JFS perhaps?

Reply:
Ubuntu won't let me format to that either. If you know how to set a label, share here.